pkg/pool-manager/pool.go

```diff
@@ -184,7 +176,7 @@ func checkRange(startMac, endMac net.HardwareAddr) error {
 		}
 	}
 
-	return fmt.Errorf("Invalid range. rangeStart: %s rangeEnd: %s", startMac.String(), endMac.String())
+	return fmt.Errorf("invalid range. rangeStart: %s rangeEnd: %s", startMac.String(), endMac.String())
```

Comment:
Boy scout mode: maybe instead of iterating manually we can use the bytes.Compare 
```golang
bytes.Compare(startMac, endMac)
```